chapters/VK_EXT_present_timing/PresentTimeInfo.txt

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) 2014-2020 Khronos Group.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+[open,refpage='VkPresentTimesInfoEXT',desc='The earliest time each image should be presented',type='structs']
+--
+
+When the `<<VK_EXT_present_timing>>` extension is enabled, additional
+fields can: be specified that allow an application to specify the earliest
+time that an image should be displayed.
+This allows an application to avoid stutter that is caused by an image being
+displayed earlier than planned.
+Such stuttering can occur with both fixed and variable-refresh-rate
+displays, because stuttering occurs when the geometry is not correctly
+positioned for when the image is displayed.
+An application can: instruct the presentation engine that an image should
+not be displayed earlier than a specified time by including the
+sname:VkPresentTimesInfoEXT structure in the pname:pNext chain of the
+sname:VkPresentInfoKHR structure.
+
+The sname:VkPresentTimesInfoEXT structure is defined as:
+
+include::{generated}/api/structs/VkPresentTimesInfoEXT.txt[]
+
+  * pname:sType is the type of this structure.
+  * pname:pNext is `NULL` or a pointer to an extension-specific structure.
+  * pname:swapchainCount is the number of swapchains being presented to by
+    this command.
+  * pname:pTimes is `NULL` or a pointer to an array of
+    sname:VkPresentTimeEXT elements with pname:swapchainCount entries.
+    If not `NULL`, each element of pname:pTimes contains the earliest time
+    to present the image corresponding to the entry in the
+    sname:VkPresentInfoKHR::pname:pImageIndices array.
+
+.Valid Usage
+****
+  * [[VUID-VkPresentTimesInfoEXT-swapchainCount-01247]]
+    pname:swapchainCount must: be the same value as
+    sname:VkPresentInfoKHR::pname:swapchainCount, where
+    sname:VkPresentInfoKHR is in the pname:pNext chain of this
+    sname:VkPresentTimesInfoEXT structure.
+****
+
+include::{generated}/validity/structs/VkPresentTimesInfoEXT.txt[]
+--
+
+[open,refpage='VkPresentTimeEXT',desc='Specifying when an image should be presented',type='structs']
+--
+
+The sname:VkPresentTimeEXT union contains either a
+slink:VkAbsolutePresentTimeEXT or a slink:VkRelativePresentTimeEXT
+structure, defined as:
+
+include::{generated}/api/structs/VkPresentTimeEXT.txt[]
+
```

Comment:
This should be described in the same way as other unions & structs, each individual member of the union getting a description following the /api/ include.